node/sc/subbridge.go

```diff
@@ -511,18 +516,45 @@ func (pm *SubBridge) handle(p BridgePeer) error {
 	for {
 		if err := pm.handleMsg(p); err != nil {
 			p.GetP2PPeer().Log().Debug("Klaytn message handling failed", "err", err)
+			return err
+		}
+	}
+}
+
+func (sc *SubBridge) resetBridgeLoop() {
+	defer sc.pmwg.Done()
+
+	ticker := time.NewTicker(resetBridgeCycle)
+	defer ticker.Stop()
 
-			if pm.peers.Len() == 1 {
-				pm.handler.setMainChainAccountNonceSynced(false)
-				atomic.StoreInt64(&pm.checkConnection, 1)
+	peerCount := 0
+	needResetSubscription := false
+
+	for {
+		select {
+		case <-sc.quitSync:
+			return
+		case <-sc.addPeerCh:
+			peerCount++
+		case <-sc.removePeerCh:
+			peerCount--
+			if peerCount == 0 {
+				needResetSubscription = true
+				sc.handler.setMainChainAccountNonceSynced(false)
+			}
+		case <-ticker.C:
+			if needResetSubscription && peerCount > 0 {
+				sc.bridgeManager.ResetAllSubscribedEvents()
```

Comment:
As I said, if `ResetAllSubscribedEvents` is failed, we can skip to reset the flag for retrying next time.

Reply:
Fixed.